src/NaiveBayes.jl

```diff
@@ -16,7 +16,10 @@ export NBModel,
         predict_proba,
         predict_logprobs,
         restructure_matrix,
-        to_matrix
+        to_matrix,
+        write_model,
+        load_model,
+        get_features_names
```

Comment:
I'm not a native speaker so definitely don't insist, but as far as I know "feature" here is an adjective and can't have plural form. For example, we say "feature vector" even though there are many features in it, "horse racing" even though there's more than one horse, etc. Yet, I may be wrong, so if you know for sure what you are doing, feel free to keep the name.